tests/runtime/conftest.py

```diff
@@ -455,7 +453,7 @@ def _check_http_server_readiness(server, config_overrides: Dict[str, Dict]):
         try:
             response = requests.get(
                 f"{mode}://localhost:{server.port}{http_server.HEALTH_ENDPOINT}",
-                verify=verify,
+                verify=False,
```

Comment:
#secure!

Reply:
This is setting up the tests, not the body of the test, so I felt it was appropriate to have this not verify and delegate that responsibility to a test that explicitly verifies.